cpp/non_iid_main.cpp

```diff
@@ -217,105 +217,101 @@ int main(int argc, char* argv[]) {
 
     // Section 6.3.1 - Estimate entropy with Most Common Value
     NonIidTestCase tc631;
+
     if (((data.alph_size > 2) || !initial_entropy)) {
         ret_min_entropy = most_common(data.bsymbols, data.blen, 2, verbose, "Bitstring", tc631);
         if (verbose == 1) printf("\tMost Common Value Estimate (bit string) = %f / 1 bit(s)\n", ret_min_entropy);
         H_bitstring = min(ret_min_entropy, H_bitstring);
+        tc631.ret_min_entropy = ret_min_entropy;
     }
 
     if (initial_entropy) {
         ret_min_entropy = most_common(data.symbols, data.len, data.alph_size, verbose, "Literal", tc631);
         if (verbose == 1) printf("\tMost Common Value Estimate = %f / %d bit(s)\n", ret_min_entropy, data.word_size);
         H_original = min(ret_min_entropy, H_original);
+        tc631.ret_min_entropy = ret_min_entropy;
```

Comment:
In cases where a test performs both literal and bitstring estimates, shouldn't both be saved? In this instance, this code saves only the literal estimate.